.github/CONTRIBUTING.md

```diff
@@ -20,32 +20,33 @@ Make sure you have a [GitHub Account](https://www.github.com)
-<!-- - [Testing Details](https://github.com/phantomcyber/phantom-apps/blob/next/.github/TESTING.md) - How we test apps & playbooks -->
+- [Coding & Conventions](https://github.com/splunk-soar-apps/.github/blob/main/.github/CONVENTIONS.md) - How we expect to see code formatted and apps named
+- [Certified vs Standard App](https://github.com/splunk-soar-apps/.github/blob/main/.github/CERTIFIED_V_UNCERTIFIED.md) definitions and differences
+- [Typical developer workflow](https://github.com/splunk-soar-apps/.github/blob/main/.github/DEV_WORKFLOW.md) - Configuring your dev environment
+<!-- - [Testing Details](https://github.com/splunk-soar-apps/.github/blob/next/.github/TESTING.md) - How we test apps & playbooks -->
 
 
 ## Step-by-Step Guide Available
-If you are not familiar with a fork-and-branch Git workflow, or just feel a bit rusty on your Git knowledge, please check out our [step-by-step contribution guide](https://github.com/phantomcyber/phantom-apps/blob/next/.github/GUIDE.md) which has actual command line examples
+If you are not familiar with a fork-and-branch Git workflow, or just feel a bit rusty on your Git knowledge, please check out our [step-by-step contribution guide](https://github.com/splunk-soar-apps/.github/blob/next/.github/GUIDE.md) which has actual command line examples
 
 
 # High Level Contribution Overview
 ## Contributing Bug-fixes
 If you've found a bug and wish to fix it, the first thing to do is
 
-1. [Fork](https://guides.github.com/activities/forking/) the project
+1. [Fork](https://guides.github.com/activities/forking/) the app repo that you
+are looking to contribute to
 1. Create a branch
 1. Make your changes on your branch
 1. Thoroughly test your changes. See the [Automated Checks](#automated-checks) section for information about basic automated checks we provide for all apps.
 1. Add your name to the contributors list in the app JSON! [Example](https://github.com/phantomcyber/phantom-apps/pull/488/commits/a02e345ce48e56bcb8711d1c5c4e40dd6e62fd11?diff=split&w=1)
-1. Open a [pull request](https://help.github.com/articles/using-pull-requests/) to the [next](https://github.com/phantomcyber/phantom-apps/tree/next) branch that gives edit access to the maintainers of the phantom-apps repo. Please ensure your pull request adheres to the guidelines mentioned in [PULL REQUEST TEMPLATE](https://github.com/phantomcyber/phantom-apps/blob/next/.github/pull_request_template.md).
+1. Open a [pull request](https://help.github.com/articles/using-pull-requests/) to the [next](https://github.com/splunk-soar-apps/phawslambda/tree/next) branch of the app repo, giving edit access to the maintainers of the repo. Please ensure your pull request adheres to the guidelines mentioned in [PULL REQUEST TEMPLATE](https://github.com/splunk-soar-apps/.github/blob/main/.github/pull_request_template.md).
```

Comment:
this probably should reference the `phawslambda` repo specifically? since this is the template

(and same thing for all the later references to the lambda app repo)

Reply:
was trying to provide an example of a ```next``` branch in an existing repo that we have under the org

it may be simpler just to remove these references though - what do you think?